CHANGELOG.md

```diff
@@ -4,9 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-- adding docker automation [#184](https://github.com/xmidt-org/caduceus/pull/184)
-- fixing caduceus queue cutoffs [#185](https://github.com/xmidt-org/caduceus/issues/185)
+## [v0.2.4]
```

Comment:
```suggestion
## [Unreleased]

## [v0.2.4]
```
We still want the unreleased header to be there

Reply:
ok I'll add that in